tests/Dfe.EarlyYearsQualification.UnitTests/Mocks/MockContentfulServiceTests.cs

```diff
@@ -191,4 +215,22 @@ public async Task GetPhaseBannerContent_ReturnsExpectedDetails()
         result.PhaseName.Should().NotBeNullOrEmpty();
         result.Show.Should().BeTrue();
     }
+    
+    [TestMethod]
+    public async Task GetCookiesBannerContent_ReturnsExpectedDetails()

```

Comment:
Suggestion (non-blocking): This test only checks that the content is populated, not what it is. Since it's being checked here anyway, it might be worth verifying that the mock service actually put each bit of content in the right place?

Reply:
Yes I think you might be right. I spotted this test was missing so added it following the way some of the others have been done but worth discussing this outside this ticket :-)